src/js/script.js

```diff
@@ -448,12 +453,44 @@
 
     remove(cartProduct){
       const thisCart = this;
+
       cartProduct.dom.wrapper.remove();
       const currentIndex = thisCart.products.indexOf(cartProduct);
       thisCart.products.splice(currentIndex, 1);
-      
       thisCart.update();
+    }
+
+    sendOrder(){
+      const thisCart = this;
+
+      const url = settings.db.url + '/' + settings.db.orders;
```

Comment:
Dodawanie stringów jest dużo wygodniej i czytelniej zapisywać przez string interpolation. W tym przypadku wyglądałoby to tak:
```suggestion
      const url = `${settings.db.url}/${settings.db.orders}`;
```